Hello Microverse Project

teke85 opened the pull request:
This Pull Request is based on the Hello Microverse Project and it fulfils the following objectives.

Create a new branch according to the GitHub flow rules.
Add a .gitignore file.
Set up a linter for HTML and CSS.
Add a descriptive README file for setup.
A .html file that includes a header with the text "Hello Microverse!".
A .css file that includes one class with styles for your header.
A test.md file with the text "This file should be ignored by git".

teke85 committed:
set up linters

teke85 committed:
add html file with header and h1 title

teke85 committed:
add css file with color blue for title

teke85 committed:
add test.md file and add to gitignore file

tresorsawasawa commented on README.md:
```diff
@@ -1,2 +1,67 @@
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```
- [ ] Kindly make you remove this broken link in your documentation as your project is not deployed to make your documentation looks professional.

teke85 replied:
Hi @tresorsawasawa . Thank you for the helpful feedback. I have removed all the broken links in the documentation.

https://github.com/teke85/Hello-Microverse/edit/Add-homepage/README.md

tresorsawasawa commented on README.md:
```diff
@@ -1,2 +1,67 @@
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- [ ] Kindly make sure you update your README documentation by filling out all sections related to your project and removing the non-related sections(e.g.: Getting started section(Prerequisites, setup, Install,...), Authors section and use your correct social links for each one, ## Acknowledgments section(fill it with updated info),...). **I suggest you use [this template for guidelines](https://github.com/MarcoAurelioAntonio/Hello-Microverse/blob/main/README.md)**

teke85 replied:
@tresorsawasawa I have used the template for guidelines and updated all the sections with updated and relevant info.

https://github.com/teke85/Hello-Microverse/edit/Add-homepage/README.md

tresorsawasawa commented on README.md:
```diff
@@ -1,2 +1,67 @@
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
- [ ] Kindly make sure you fix this broken link in the **License section** of the README file, **Line 37** (`[MIT](./MIT.md)`) to make it more professional. I think you can fix this by only adding this [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) license file in your root directory and edit it according to your information.

teke85 replied:
@tresorsawasawa I fixed the MIT license file and edited it accordingly. 
![Hello-Microverse-MIT-md-at-Add-homepage-·-teke85-Hello-Microverse](https://user-images.githubusercontent.com/29442846/176606132-c340c1b4-d9ce-45e3-ac7d-a15d12e70a63.png)

tresorsawasawa commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
- [ **OPTIONAL** ] I suggested you also add this `.vscode/` directory in your `.gitignore` file as it contains nonrelated files of the project so it can be ignored by Git and not appear on GitHub.

teke85 replied:
@tresorsawasawa I have added .vscode/ directory in my .gitignore file

![Hello-Microverse-gitignore-at-Add-homepage-·-teke85-Hello-Microverse](https://user-images.githubusercontent.com/29442846/176613741-c1d9b440-abcc-45a4-9272-162a277a2bb1.png)

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @teke85 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

teke85 committed:
remove broken links and make appropriate changes to ReadMe file

I removed all the broken links and add more information about the project to the template were appropriate

teke85 committed:
add .vscode to gitignore file

teke85 committed:
add MIT file to root directory

added MIT file to the root directory of my folder

teke85 committed:
change broken social media links

teke85 committed:
fix github link

teke85 committed:
add link to MIT License

teke85 committed:
add copyright info

teke85 committed:
Update MIT.md

teke85 committed:
Update .gitignore

Bondok6 commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
- [OPTIONAL] I can see that you already added the `.vscode/` folder to .gitignore. However, the file is still present in your GitHub repository !! to Solve this small problem I suggest you to [Check this](https://medium.com/sela-devops-team/gitignore-doesnt-work-e1566845d30a). Or you can delete it manually from GitHub (another solution).

Bondok6 reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

## Hi @teke85 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great work on making the changes requested by a previous reviewer 👌
- Your project is professional ✅
- Your repo is professional ✅
-  Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

<img src="https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966" />


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
